internal/runners/publish/publish.go

```diff
@@ -247,7 +247,7 @@ func (r *Runner) Run(params *Params) error {
 	result := graphModel.PublishResult{}
 
 	if err := r.client.Run(pr, &result); err != nil {
-		return locale.WrapError(err, "err_uploadingredient_publish", "Could not publish ingredient")
+		return locale.WrapInputError(err, "err_uploadingredient_publish", "", err.Error())
```

Comment:
We can't just assume all publish errors are input errors. But we should be able to identify this from the http status code returned.

Reply:
That's fair. I tried manipulating our graphql query, but discovered that the `publish` endpoint is not returning an expected error type. It's just returning a list of errors. I've filed https://activestatef.atlassian.net/jira/software/c/projects/PB/issues/PB-4889 and https://activestatef.atlassian.net/browse/DX-2592 to address this. Until then, at least we'll have a more informative error message to present the user with.